test_reporting/junit_xml_parser.py

```diff
@@ -422,7 +422,8 @@ def _parse_test_case(test_case):
             result[attribute] = test_case.get(attribute)
         for attribute in REQUIRED_TESTCASE_PROPERTIES:
             testcase_properties = _parse_testcase_properties(test_case)
-            result[attribute] = testcase_properties[attribute]
+            if testcase_properties[attribute]:
```

Comment:
Do you mean `if attribute in testcase_properties:`?
If key `attribute` does not exist in `testcase_properties`, statement `testcase_properties[attribute]` still will raise `KeyError`.

Reply:
Oh I see, my bad